lnrpc/routerrpc/router.proto

```diff
@@ -348,6 +362,11 @@ message RouteFeeResponse {
     value.
     */
     int64 time_lock_delay = 2;
+
+    /*
+    An indication whether a probing payment reached the target node.
+     */
+    bool dest_reachable = 3;
```

Comment:
shouldn't the fee be returned here?

Reply:
yes, the `routing_fee_msat` was already part of this message, see https://github.com/lightningnetwork/lnd/pull/8136/files/a0e152261696bef4197f7aaa7ed9068295b334f4#diff-35316b62bc12f39d4bf2426add7562a564d7deee49b9f9644f42766abb672c80R357